ports/meta/macathena-extra-software-nox/debathena-list

```diff
@@ -0,0 +1,26 @@
+# the remainder of the debathena-extra-software-nox list, that isn't in macathena-extra-software
+Source: debathena-extra-software-nox
+Section: debathena/base
+Priority: extra
+Standards-Version: 3.9.3
+Maintainer: Debathena Project <debathena@mit.edu>
+Build-Depends: cdbs, debhelper, dh-buildinfo
+
+Package: debathena-extra-software-nox
+Architecture: all
+Depends: build-essential, debathena-debian-dev, # I think these are debian specific
```

Comment:
They are, but you should install the equivalent packages for MacPorts. `build-essential` is just a metapackage. I think this might boil down to:

- `dpkg`
- `debianutils`
- `gmake`

since we should already have a compiler.